src/Uno.UI/Microsoft/UI/Xaml/Controls/InfoBar/Strings/en-us/Resources.resw

```diff
@@ -0,0 +1,140 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
Since I just browed this in WinUI I can answer. Yes, only en-US strings are available.  It's almost always a separate commit after the translation team is done to add the others. Sometimes this can take several weeks it seems and this control is brand-new. Martin is too quick!

https://github.com/microsoft/microsoft-ui-xaml/tree/master/dev/InfoBar

Reply:
Yes, so far there are no translations :-D . I think they will be added soon though.